src/components/TeachersPage/LessonForm/LessonForm.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+
+const LessonForm: React.FC = () => {
+  const [formData, setFormData] = useState({
+    fullName: '',
+    email: '',
+    phone: '',
+    preferences: {
+      careerAndBusiness: false,
+      lessonForKids: false,
+      livingAbroad: false,
+      examsAndCoursework: false,
+      cultureTravelHobby: false
+    }
+  })
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target
+    setFormData((prev) => ({
+      ...prev,
+      [name]: value
+    }))
+  }
+
+  const handleCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, checked } = e.target
+    setFormData((prev) => ({
+      ...prev,
+      preferences: {
+        ...prev.preferences,
+        [name]: checked
+      }
+    }))
+  }
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+    console.log('Form Data:', formData)
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label>
+          <input
+            type="checkbox"
+            name="careerAndBusiness"
+            checked={formData.preferences.careerAndBusiness}
+            onChange={handleCheckboxChange}
+          />
+          Career and business
+        </label>
+        <label>
+          <input
+            type="checkbox"
+            name="lessonForKids"
+            checked={formData.preferences.lessonForKids}
+            onChange={handleCheckboxChange}
+          />
+          Lesson for kids
+        </label>
+        <label>
+          <input
+            type="checkbox"
+            name="livingAbroad"
+            checked={formData.preferences.livingAbroad}
+            onChange={handleCheckboxChange}
+          />
+          Living abroad
+        </label>
+        <label>
+          <input
+            type="checkbox"
+            name="examsAndCoursework"
+            checked={formData.preferences.examsAndCoursework}
+            onChange={handleCheckboxChange}
+          />
+          Exams and coursework
+        </label>
+        <label>
+          <input
+            type="checkbox"
+            name="cultureTravelHobby"
+            checked={formData.preferences.cultureTravelHobby}
+            onChange={handleCheckboxChange}
+          />
+          Culture, travel or hobby
+        </label>
```

Comment:
check folder UI-components, custom input is ready

Reply:
same